Project/src/main/resources/templates/editor.html

```diff
@@ -21,10 +21,10 @@
     <table class="table table-bordered" id="editor_table">
         <thead class="thead-dark">
             <th>File Name</th>
-            <th>Reviewer</th>
-            <th>Date</th>
+            <th class="no-sort">Reviewer</th>
+            <th class="no-sort">Date</th>
```

Comment:
Should dates not also be sortable?

Reply:
no

![image](https://user-images.githubusercontent.com/15215413/54566428-2e7fbc00-49a7-11e9-9488-5bbee92281f3.png)